core/templates/pages/exploration-editor-page/services/changes-in-human-readable-form.service.spec.ts

```diff
@@ -17,7 +17,7 @@
  */
 
 import { ChangesInHumanReadableFormService } from
-  'pages/exploration-editor-page/services/changes-in-human-readable-form.service.ts';
+  'pages/exploration-editor-page/services/changes-in-human-readable-form.service.ts'; // eslint-disable-line
```

Comment:
Nit: Can we split this string in multiple lines?

Reply:
I don't think so. Simply concatenating strings like this:
```typescript
import { ChangesInHumanReadableFormService } from
  'pages/exploration-editor-page/services/changes-in-human-readable-form.' +
  'service.ts';
```
didn't work for me. Is there another way that I'm missing?

Reply:
Oh okay, I think this is fine in that case. Thanks for checking!